### PR TITLE
Feat(implementation): create news page and components

### LINKED ISSUE #30: Create news page and components
Create the base components for news page and basic css layout. Also includes the subcategory pages for news.

### PR DESCRIPTION
**Pull request by JDinis88**
## Changes
1. Added main news page
2. Created template for news sub pages
3. Added routing for the pages, and updated navbar to reflect changes
4. Small staff page fixes

## Purpose
Major implementation requirement

## Approach
Two similar components, one for main, and one for sub pages. Share CSS file

_If this closes an issue, reference the issue here. If it doesn't, remove this line_
Closes #030

### CODE REVIEW

File: client/src/pages/NewsPage.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.subpage--secondary-articles .article-thumbnail--subheading,.article-thumbnail--author, .article-thumbnail--body{
+    display:none;
+}
+
+
+.subpage--article-carousel-header{
+    padding-top: 1rem;
+}
+.subpage--article-carousel-container{
+    border-bottom: .0625rem solid var(--element-2-color, gray);
```
Comment (tkgonzal) [inline client/src/pages/NewsPage.css:46]: praise: love me some root variables

File: client/src/pages/StaffPage.css [hunk 2]
```diff
@@ -1,17 +1,18 @@
 .staff-page{
-    width: 100%;
+    width: 100%-25rem;
```
Comment (tkgonzal) [inline client/src/pages/StaffPage.css:2]: nitpick: i think in general when you do a calc like this its better to put it in a calc function. like
```
width: calc(100% - 25rem);
```
just to for safety when being used on different browsers.

**Review by tkgonzal — APPROVED** [review]
praise: Looks good ! Left some comments

**Review by missile720 — APPROVED** [review]
(praise): Looks good!

**Pull request merged**